### PR TITLE
Automated linting

### PR DESCRIPTION
**Pull request by bcongdon**
Adds automated linting via ESLint.

I think it'd be good to have linting standards in this repo, especially because of how large it's gotten.
I'm not particularly set on the specific linting rules (open to input), but I do think it'd be good to have standards across all the controllers.

You can run the linter with `npm run lint`

### CODE REVIEW

**Comment by aashishkapur** [comment]
Yeah, definitely good to have. Probably a good idea to bring this to the other node services too

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "node": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 6,
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            2
```
Comment (sameetandpotatoes) [inline .eslintrc.js]: This was driving me crazy for so long

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+module.exports = {
```
Comment (sameetandpotatoes) [inline .eslintrc.js]: Does this enforce the `const` thing for require statements?

Comment (bcongdon) [inline .eslintrc.js]: Unfortunately, no. The closest thing is [prefer-const](http://eslint.org/docs/rules/prefer-const), but that doesn't really do what we want.

**Pull request merged**